oceannavigator/__init__.py

```diff
@@ -39,7 +39,7 @@ def config_dask(app) -> None:
     )
 
 
-def create_app(testing: bool = False):
+def create_app(testing: bool = False, dataset_config_path: str = ""):
```

Comment:
Pass the dataset config path we want to load in here so we don't have to mock this every time in the tests! Too much mocking going on atm.